build_tools/buildkite/cmake/android/arm64-v8a/pipeline.yml

```diff
@@ -25,7 +25,7 @@ steps:
       - "buildkite-agent artifact download --step build build-artifacts.tgz ./"
       - "tar xzf build-artifacts.tgz"
       - "find build-android/ -name '*.cmake' -exec sed -i \"s!\\$IREE_DOCKER_WORKDIR/!\\$PWD/!g\" {} \\;"
-      - "ctest --test-dir build-android/ --timeout 900 --output-on-failure"
+      - "ctest --test-dir build-android/ --timeout 900 --output-on-failure --no-tests=error"
```

Comment:
intentional merge conflict in here to test the prior behavior: https://buildkite.com/iree/iree-test-android/builds/6410
I'll resolve that once I verify that this catches the misconfiguration issue

Reply:
Aaaaand the tests succeeded with the `--no-tests=error` flag ignored. I think we'll actually need to update CMake on those lab devices to get any of these benefits.

@GMNGeoffrey or @antiagainst can one of you help with that?

Reply:
(And while you're in there, maybe check `cmake --version`... I'm curious how out date they are...)

Reply:
:facepalm:

Reply:
I was going to ask why the CMAKE_VERSION in our config didn't cause this to fail, but I guess since we build on another machine, that doesn't apply...

Reply:
Gah, now that CMake is updated, `--test-dir build-android/` is handled correctly, so tests ran on my trial run :P
I'm pretty confident that `--no-tests=error` will work too, but it would be nice to get confirmation from the CI...

Reply:
You could drop `--test-dir` and see what happens